src/dlgIRC.cpp

```diff
@@ -662,7 +666,9 @@ void dlgIRC::slot_joinedChannel(IrcJoinMessage* message)
 
     if (message->isOwn()) {
         const QString luaText = IrcMessageFormatter::formatMessage(static_cast<IrcMessage*>(message), true);
-        mpHost->postIrcMessage(message->nick(), message->channel(), luaText);
+        if (mpHost) {
```

Comment:
Why not just bail out at the start of the method if `mpHost` is a `nullptr` there?